File: String/709_ToLowerCase/ToLowerCase.php
class Solution {
	/**
	 * @param String $str
	 * @return String
	 */
	function toLowerCase($str) {
		for ($i=0;$i<strlen($str);$i++) {
			$value = ord($str[$i]);
			if ($value >= ord('A') && $value <= ord('Z')) {
				$str[$i] = chr($value +32);
			}
		}
		return $str;
	}
}

File: String/709_ToLowerCase/ToLowerCase.py
class Solution(object):
	def toLowerCase(self, str):
		"""
		:type str: str
		:rtype: str
		"""
		res = ""
		for ch in res:
			if ord('A') <= ord(ch) <= ord('Z'):
				res += chr(ord(ch) + 32)
			else:
				res += ch
		return res
